Makefile

```diff
@@ -1,22 +1,60 @@
-TOOLS = docker-compose run --rm tools
-CONSOLE = ./bin/console
+DOCKER:=$(shell docker-compose -v dot 2> /dev/null)
 
-all: boot install assets run
+ifdef DOCKER
+    APP=docker-compose exec -T app
+    TOOLS=docker-compose run --rm tools
+endif
 
-boot:
+CONSOLE=$(APP) bin/console
+
+.PHONY: clean
+
+all: start dependencies db perm
+
+dependencies: composer yarn assets perm
+
+stop:
+ifdef DOCKER
+	docker-compose down
+endif
+
+start:
+ifdef DOCKER
 	docker-compose up -d
+else
+	bin/console server:run
```

Comment:
Yes, done, as another form in my last push. Starting the server from Symfony was a bad idea when Composer was not been executed before...